utoipa-gen/src/schema.rs

```diff
@@ -107,13 +118,20 @@ impl<'a> ComponentPart<'a> {
         generic_component_type
     }
 
-    fn get_generic_arg_type(index: usize, args: &'a AngleBracketedGenericArguments) -> &'a Type {
+    fn get_generic_arg_type(
+        index: usize,
+        args: &'a AngleBracketedGenericArguments,
+        skip_first: bool,
```

Comment:
Pretty nifty solution this `skip_first`. I originally thought more complicated solution but this actually is fine too. It can be enhanced later. I was considering refactoring this a bit anyways.